### ISSUE TITLE
Fixed time step config

### ISSUE THREAD

**Issue by bentles** [ChrisPritchard/Xelmish#19]
Is it possible to make the fixed timestep configurable? I can add it myself just not sure if xelmish relies on the timestep being fixed in some way

**Comment by ChrisPritchard** [comment]
No xelmish doesnt rely on it, and neither should you I would think? Almost always doing calculations with elapsed game time is more appropriate. Whatcha need to change it for?

**Comment by bentles** [comment]
I was thinking if you set the fixed timestep (and vsync I think) to false you can have the game draw as fast as possible and then implement a fixed timestep inside of that loop [like this](https://gafferongames.com/post/fix_your_timestep/) for the physics. I guess another option is to set the TargetElapsedTime to a smaller value (for a higher frame rate) and then skip calls to update the physics at a lower frame rate.

**Comment by ChrisPritchard** [comment]
Right. My understanding is that drawing faster might cause some stuttering due to garbage collection, but feel free to create a PR and I'll take a look. Or you can fork the project and have a go if you like?

**Comment by bentles** [comment]
Cool I'll give it a go after work today
